roles/setup_gitops/tasks/local-registry.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Assert that a path for the pull_secret is defined
+  ansible.builtin.assert:
+    that:
+      - sg_pullsecret_file | default('') | length > 0
```

Comment:
We can move this validations to meta/argument_specs.yml, see: https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/711/files#diff-f2517c843712247cd9fe949ed7de980c73b889347466af2909c18bfa2b529214

Reply:
This is required if sg_local_registry must be defined, and I think such dependencies are not possible in arg_specs.